src/components/SignUpForm/index.js

```diff
@@ -34,35 +39,76 @@ const SignUpForm = ({ onSubmit }) => {
   const inputProps = useTextInputProps(handleValueChange, handleBlur, values);
 
   return (
-    <>
+    <View style={styles.container}>
+      <Input
+        label={strings.SIGN_UP.name}
+        testID="name-input"
+        error={errors.name}
+        invalid={error || errors.name}
```

Comment:
Also, you can use the error prop for this, there is no need to define this new one. This comment applies for all fields